packages/oracle/src/_internal/connection-options.ts

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved
+
+import type { PickByType } from '@sequelize/utils';
+import { getSynchronizedTypeKeys } from '@sequelize/utils';
+import type { OracleConnectionOptions } from '../connection-manager.js';
+
+type StringConnectionOptions = PickByType<OracleConnectionOptions, string>;
+
+const STRING_CONNECTION_OPTION_MAP = {
+  configDir: undefined,
+  connectionIdPrefix: undefined,
+  connectString: undefined,
+  connectionString: undefined,
+  database: undefined,
+  debugJdwp: undefined,
+  edition: undefined,
+  host: undefined,
+  httpsProxy: undefined,
+  newPassword: undefined,
+  password: undefined,
+  poolAlias: undefined,
+  port: undefined,
+  sourceRoute: undefined,
+  sslServerCertDN: undefined,
+  tag: undefined,
+  user: undefined,
+  username: undefined,
```

Comment:
Same here. Imo as these are aliases of each-other, we should only allow one of them